src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -63,9 +93,271 @@ private MobileAdsDataProvider() {
+                "\"age\":[\"teen\",\"kid\",\"18-34\"]",
+                "\"gnre\":[\"anime\",\"fantasy\",\"strategy\",\"comics\"]",
+                "\"media\":[\"tv\",\"cards\"]",
+                "\"pform\":[\"pc\",\"psp\"]",
+                "\"sex\":[\"m\"]"
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        },
+        {
+            "fallout",
+            "Fallout",
+            "wka.gaming/_fallout//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"gaming\"",
+                "\"s0v\":\"games\"",
+                "\"s0c\":[\"gaming\"]",
+                "\"s1\":\"_fallout\"",
+                "\"s2\":\"article\"",
+                "\"ar\":\"4:3\"",
+                "\"artid\":\"948\"",
+                //"\"cat\":[\"fallout\"]",
+                "\"dmn\":\"wikiacom\"",
+                "\"hostpre\":\"fallout\"",
+                "\"ab\":[",
+                "\"ksgmnt\":[",
+                "\"u\":\"",
+                "\"dmn\":\"",
+                "\"skin\":\"mercury\"",
+                "\"lang\":\"en\"",
+                "\"wpage\":\"fallout\"",
+                "\"ref\":\"direct\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"sex\":[\"m\"]",
+                "\"age\":[\"under18\",\"18-34\",\"18-24\",\"25-34\",\"teen\"]",
+                "\"gnre\":[\"action\",\"adventure\",\"fps\",\"openworld\",\"rpg\",\"scifi\"]",
+                "\"pform\":[\"pc\",\"xbox360\",\"ps3\"]",
+                "\"pub\":[\"bethesda\",\"bethesda\"]",
+                "\"esrb\":[\"mature\"]",
+                "\"theme\":[\"military\",\"postapocalypse\"]"
+
+            ),
+            Arrays.asList(
+                "\"pos\":\"MOBILE_TOP_LEADERBOARD\"",
+                "\"src\":\"mobile\""
+            )
+        },
+        {
+            "runescape",
+            "Grew",
+            "wka.gaming/_runescape//article",
+            "MOBILE_TOP_LEADERBOARD",
+            Arrays.asList(
+                "\"s0\":\"gaming\"",
+                "\"s0v\":\"games\"",
+                "\"s0c\":[\"gaming\"]",
+                "\"s1\":\"_runescape\"",
+                "\"s2\":\"article\"",
+                "\"ar\":\"4:3\"",
+                "\"hostpre\":\"runescape\"",
+                "\"ab\":[",
+                "\"dmn\":\"",
+                "\"skin\":\"mercury\"",
+                "\"artid\":\"33150\"",
+                "\"wpage\":\"grew\"",
+                "\"ref\":\"direct\"",
+                "\"lang\":\"en\"",
+                "\"pv\":\"1\"",
+                "\"top\":\"1k\"",
+                "\"age\":[\"kids\",\"teen\",\"under18\",\"18-24\"]",
+                "\"gnre\":[\"mmo\",\"rpg\",\"action\",\"adventure\",\"free2play\",\"fantasy\",\"mmo\",\"rpg\",\"mmorpg\",\"openworld\"]",
+                "\"pform\":[\"pc\"]",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "\"pform\":[\"pc\"]" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)